### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by esteve**
This PR fixes style errors in the latest master

### CODE REVIEW

**Comment by esteve** [comment]
@bergercookie @nnmm can you have a look at this PR when you have a moment? Thanks!

**Review by nnmm — APPROVED** [review]
Sure.

**Comment by esteve** [comment]
@nnmm thanks!

File: test/test_spell_check.py [hunk 1]
```diff
@@ -25,8 +25,7 @@ def test_spell_check():
             '--enable=spelling',
             '--spelling-dict=en_US',
             '--ignore-comments=no',
-            '--spelling-private-dict-file=' +
-            str(spell_check_words_path),
+            '--spelling-private-dict-file=' + str(spell_check_words_path),
```
Comment (bergercookie) [inline test/test_spell_check.py:28]: nit: I'd use `f'--spelling-private-dict-file={spell_check_words_path}'.
It's both more readable and faster than string addition

**Pull request merged**